Documentation/installation/k0s.rst

```diff
@@ -0,0 +1,99 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k0s_install:
+
+*****************************
+Installation k0s Using k0sctl
+*****************************
+
+This guide walks you through installation of Cilium on `k0s <https://k0sproject.io/>`_,
+an open source, all-inclusive Kubernetes distribution, 
+which is configured with all of the features needed to build a Kubernetes cluster.
+
+Cilium is presently supported on amd64 and arm64 architectures.
+
+Install a Master Node
+=====================
+
+The first step (after installing the k0sctl binary obviously) is to prepare the yaml configuration file k0sctl will use:
+
+.. code-block:: shell-session
+
+   # The following command assumes the user has deployed 3 VMs
+   # with the default user "k0s" using the default ssh-key (without passphrase)
```

Comment:
This seems like something that should be an explicit step, something like "Set up your VMs. How to do this is out of scope of this guide. After deploying the VMs, take note of the IP addresses of the VMs on the local network. These will be used in a later step.".

You could even get the reader to define an environment variable with these here, then later on in the k0sctl command just replace the current hard-coded IP addresses with the variable.